examples/multimodal/multimodal.ts

```diff
@@ -0,0 +1,40 @@
+import https from 'https'
```

Comment:
Let's just load a file from disk in the repo (e.g. the watermelon bike or astronaut on horse) and show – the key idea here isn't to show a full app, but something quick to understand and copy and paste

Reply:
Used a local cat image instead